docs/quick-start.md

```diff
@@ -1,31 +1,52 @@
 # Quick Start
 
-To see how Argo Workflows work, you can install it and run examples of simple workflows and workflows that use artifacts.
+To see how Argo Workflows work, you can install it and run examples of simple workflows.
 
-Before you start you need a Kubernetes cluster and `kubectl` set-up
+Before you start you need a Kubernetes cluster and `kubectl` set-up to be able to access that cluster.
```

Comment:
Nit: I'd change "set-up" => "set up" while we're in  here

Reply:
Also, I think this is a good place to link out to the common local K8s deployment tools: minikube/kind/k3d/etc. 

I'm thinking of users coming here without a lot of K8s experience. It could help to ensure they are directly linked to a tool for this, or at least know that all of these work fine for Quickstart.

Local K8s cluster options:
- [MiniKube](https://minikube.sigs.k8s.io/docs/)
- [kind](https://kind.sigs.k8s.io/)
- [K3s](https://rancher.com/docs/k3s/latest/en/quick-start/) or [K3d](https://k3d.io/)
- [Docker Desktop](https://docs.docker.com/desktop/kubernetes/)